test/boost/cql_query_test.cc

```diff
@@ -4313,3 +4315,207 @@ SEASTAR_TEST_CASE(test_rf_expand) {
+                "    AND dclocal_read_repair_chance = 0.2\n"
+                "    AND default_time_to_live = 0\n"
+                "    AND gc_grace_seconds = 954000\n"
+                "    AND max_index_interval = 3048\n"
+                "    AND memtable_flush_period_in_ms = 10\n"
+                "    AND min_index_interval = 128\n"
+                "    AND read_repair_chance = 0\n"
+                "    AND speculative_retry = '99.0PERCENTILE';\n"
+            },
+            {"Cf3", "CREATE TABLE ks.\"Cf3\" (\n"
+                "    pk blob,\n"
+                "    pk2 blob,\n"
+                "    \"CK\" blob,\n"
+                "    col1 blob,\n"
+                "    col2 blob,\n"
+                "    PRIMARY KEY ((pk, pk2), \"CK\")\n"
+                ") WITH CLUSTERING ORDER BY (\"CK\" DESC)\n"
+                "    AND bloom_filter_fp_chance = 0.02\n"
+                "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+                "    AND comment = ''\n"
+                "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+                "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+                "    AND crc_check_chance = 1\n"
+                "    AND dclocal_read_repair_chance = 0.2\n"
+                "    AND default_time_to_live = 0\n"
+                "    AND gc_grace_seconds = 954000\n"
+                "    AND max_index_interval = 3048\n"
+                "    AND memtable_flush_period_in_ms = 10\n"
+                "    AND min_index_interval = 128\n"
+                "    AND read_repair_chance = 0\n"
+                "    AND speculative_retry = '99.0PERCENTILE';\n"
+            },
+            {"cf4", "CREATE TABLE ks.cf4 (\n"
+                "    pk blob,\n"
+                "    col1 blob,\n"
+                "    col2 blob,\n"
+                "    pn phone,\n"
+                "    PRIMARY KEY (pk)\n"
+                ") WITH "
+                "     bloom_filter_fp_chance = 0.02\n"
+                "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+                "    AND comment = ''\n"
+                "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+                "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+                "    AND crc_check_chance = 1\n"
+                "    AND dclocal_read_repair_chance = 0.2\n"
+                "    AND default_time_to_live = 0\n"
+                "    AND gc_grace_seconds = 954000\n"
+                "    AND max_index_interval = 3048\n"
+                "    AND memtable_flush_period_in_ms = 10\n"
+                "    AND min_index_interval = 128\n"
+                "    AND read_repair_chance = 0\n"
+                "    AND speculative_retry = '99.0PERCENTILE';\n"
+            }
+
+        };
+        e.execute_cql("CREATE TYPE phone ("
```

Comment:
From what I've tested, if the type is quoted-capitalized the type will be created, but the create table will fail because it will still look for the lower case name.

Reply:
So, the code is wrong, right (I left you a comment exactly where I think the code is wrong)? So why not fix it? And why not add the test here to confirm the code has been fixed?